cmd/thanos/main.go

```diff
@@ -355,24 +356,27 @@ func metricHTTPListenGroup(g *run.Group, logger log.Logger, reg *prometheus.Regi
 
 // defaultHTTPListener starts a run.Group that servers HTTP endpoint with default endpoints providing Prometheus metrics,
```

Comment:
Plus a small typo here so if you will rename it then you could fix it as well :smile: